R/si.R

```diff
@@ -132,14 +132,16 @@ si.stanreg <- function(posterior, prior = NULL,
                        BF = 1, verbose = TRUE,
```

Comment:
Looks all good to me! Was just wondering if we should name the argument `BF`, or `bayesfactor`, or both (alias)?

Reply:
Having `bayesfactor` might be more confusing than it already is, as this "BF" is not explicitly ***the*** Bayes factor, but a Bayesian (updating) factor. So I think keeping it as is (`BF`) is best, which also matches the terminology in the SI paper (where the support level is simply rendered as *BF*).

Reply:
ok, sound feasible!

Reply:
~So what you're saying is that `F_to_eta2` should be `F_to_eta2_partial`?~
👍